src/eth/GasEstimatorService.js

```diff
@@ -3,32 +3,42 @@ import { PublicService } from '@makerdao/services-core';
 export default class GasEstimatorService extends PublicService {
   constructor(name = 'gasEstimator') {
     super(name, ['web3', 'log']);
-    this._percentage = null;
+    this._percentage = 1.55;
     this._absolute = null;
+    this._fallback = 4000000;
   }
 
-  estimateGasLimit(transaction) {
-    if (this._percentage === null && this._absolute === null) {
-      throw new Error('no gas limit policy set');
+  async estimateGasLimit(transaction, options = {}) {
+    let web3Data = [];
+    try {
+      web3Data = await Promise.all([
+        this.get('web3').getBlock('latest'),
+        this.get('web3').estimateGas(transaction)
+      ]);
+    } catch (err) {
+      options.gasLimit = this._fallback;
+      return options;
     }
 
-    return Promise.all([
-      this.get('web3').getBlock('latest'),
-      this.get('web3').estimateGas(transaction)
-    ]).then(web3Data => {
-      const blockLimit = web3Data[0].gasLimit,
-        estimate = web3Data[1];
-
-      if (this._percentage === null && this._absolute !== null) {
-        return Math.min(this._absolute, blockLimit);
-      }
+    const blockLimit = web3Data[0].gasLimit;
+    const estimate = web3Data[1];
 
-      if (this._absolute === null) {
-        return Math.min(estimate * this._percentage, blockLimit);
-      }
+    if (this._percentage === null && this._absolute !== null) {
```

Comment:
if you know that 0 is not a valid value for these variables, then there's no need to explicitly check equality with null -- just check truthiness or falsiness (i.e. `if (!this._percentage && this._absolute)`)

Reply:
Good call--this and the get/set functions were already in this service, they just weren't working